Products/CMFPlone/CatalogTool.py

```diff
@@ -425,6 +427,21 @@ def indexObject(obj, path):
                     safe_callable(obj.indexObject)):
                 try:
                     obj.indexObject()
+
```

Comment:
We're cleaning up the monkey patch, not refactoring code as this stage. 

Furthermore we can't use plone.api in Products.CMFPlone (circular dependencies)